### DISCUSSION TITLE
infrastructure/ubiquiti/udm-dream-machine/udm-better-fan-speeds/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#91] [General]
# infrastructure/ubiquiti/udm-dream-machine/udm-better-fan-speeds/

Overrides UDM fan speeds with users predefined

https://3os.org/infrastructure/ubiquiti/udm-dream-machine/udm-better-fan-speeds/

**Comment by DumSkidderik** [comment]
Excellent, sitting next to my UDM Pro is now bearable again :)

**Comment by ChrisProAU** [comment]
Install does not work 
chmod: cannot access '/mnt/data/on_boot.d/11-udm-better-fan-speed.sh': No such file or directory

**Reply by fire1ce** [reply]
Hey, since the v2.x, v3.x. The fans curve and the logic been revised by Unifi's firmware. I'll be archiving this repo, there no more need in it.
